Controls/TexturePicker.xaml

```diff
@@ -64,8 +64,40 @@
                                                         <ColumnDefinition Width="9*" />
                                                         <ColumnDefinition Width="Auto" />
                                                     </Grid.ColumnDefinitions>
-                                                    <Button Content="X" Grid.Column="1" Width="20" Command="{Binding RelativeSource={RelativeSource AncestorType={x:Type local:TexturePicker}},
-                                              Path=DataContext.Textures.RemoveCommand}" CommandParameter="{Binding}"></Button>
+                                                    <Button Grid.Column="1" Width="20" Cursor="Hand" Command="{Binding RelativeSource={RelativeSource AncestorType={x:Type local:TexturePicker}}, Path=DataContext.Textures.RemoveCommand}" CommandParameter="{Binding}">
+                                                        <Button.Style>
+                                                            <Style TargetType="{x:Type Button}">
+                                                                <Setter Property="Template">
+                                                                    <Setter.Value>
+                                                                        <ControlTemplate TargetType="Button">
+                                                                            <Border>
+                                                                                <Grid Background="Transparent">
+                                                                                    <ContentPresenter/>
+                                                                                </Grid>
+                                                                            </Border>
+                                                                        </ControlTemplate>
+                                                                    </Setter.Value>
+                                                                </Setter>
+                                                                <Setter Property="Content">
+                                                                    <Setter.Value>
+                                                                        <Path Fill="DarkRed" Stretch="Uniform"  Data="{DynamicResource Delete}"/>
```

Comment:
the texture picker button "Delete" uses DarkRed twice but the top one should be Red, I may have saved it incorrectly.
or if you like DarkRed more, please make the lower black

Reply:
Ahh.  I see what you mean.  Yes I like that.  Let me add it back

Reply:
looks really good 👍